doc/classes/ArrayMesh.xml

```diff
@@ -272,14 +298,44 @@
 		<constant name="ARRAY_FORMAT_TEX_UV2" value="32" enum="ArrayFormat">
 			Array format will include another set of UVs.
 		</constant>
-		<constant name="ARRAY_FORMAT_BONES" value="64" enum="ArrayFormat">
+		<constant name="ARRAY_FORMAT_CUSTOM0" value="64" enum="ArrayFormat">
+		</constant>
+		<constant name="ARRAY_FORMAT_CUSTOM1" value="128" enum="ArrayFormat">
+		</constant>
+		<constant name="ARRAY_FORMAT_CUSTOM2" value="256" enum="ArrayFormat">
+		</constant>
+		<constant name="ARRAY_FORMAT_CUSTOM3" value="512" enum="ArrayFormat">
+		</constant>
+		<constant name="ARRAY_FORMAT_BONES" value="1024" enum="ArrayFormat">
 			Array format will include bone indices.
 		</constant>
-		<constant name="ARRAY_FORMAT_WEIGHTS" value="128" enum="ArrayFormat">
+		<constant name="ARRAY_FORMAT_WEIGHTS" value="2048" enum="ArrayFormat">
 			Array format will include bone weights.
 		</constant>
-		<constant name="ARRAY_FORMAT_INDEX" value="256" enum="ArrayFormat">
+		<constant name="ARRAY_FORMAT_INDEX" value="4096" enum="ArrayFormat">
 			Index array will be used.
 		</constant>
+		<constant name="ARRAY_FORMAT_BLEND_SHAPE_MASK" value="-8185" enum="ArrayFormat">
```

Comment:
@reduz Overflowing `int32_t`?

Reply:
Same stuff on `Mesh`. BTW shouldn't we bind those only in `Mesh` and not `ArrayMesh`? Aren't they exactly the same?

Reply:
> Overflowing `int32_t`?

reduz will fix that in a follow-up PR.

> BTW shouldn't we bind those only in Mesh and not ArrayMesh? Aren't they exactly the same?

I fixed that.